app/models/referenceData/Country.scala

```diff
@@ -14,18 +14,29 @@
  * limitations under the License.
  */
 
-package models
+package models.referenceData
 
 import cats.Order
-import models.referenceData.RichComparison
-import play.api.libs.json.{Json, OFormat}
+import config.FrontendAppConfig
+import play.api.libs.functional.syntax.*
+import play.api.libs.json.{__, Json, Reads}
 
 case class Country(code: String, description: String) {
   override def toString: String = s"$description - $code"
 }
 
 object Country {
-  implicit val format: OFormat[Country] = Json.format[Country]
+
+  def reads(config: FrontendAppConfig): Reads[Country] =
+    if (config.phase6Enabled) {
+      (
+        (__ \ "key").read[String] and
+          (__ \ "value").read[String]
+      )(Country.apply)
+    } else {
+      Json.reads[Country]
+    }
 
```

Comment:
@jackheslop96 correct me if I am wrong, we will need an implicit "writes" method here for each case class right? cc. @Roshangautam2052

Reply:
No writes needed in this particular microservice @justega247 . We don't write the data to mongo or anything like that. We only read the data in from customs-reference-data.